pkgs/development/python-modules/psutil/default.nix

```
@@ -43,15 +43,15 @@ buildPythonPackage rec {
   # https://github.com/giampaolo/psutil/issues/1715
   doCheck = !stdenv.hostPlatform.isDarwin;
 
+  # Note: $out must be referenced as test import paths are relative
+  preCheck = ''
+    appendToVar pytestFlagsArray "$out/${python.sitePackages}/psutil/tests/test_system.py"
```

Why not use
```suggestion
    appendToVar pytestFlagsArray "${placeholder "out"}/${python.sitePackages}/psutil/tests/test_system.py"
```
?
Can we can keep using `pytestFlagsArray` with that change?

---

The commit message should be reworded.